src/generated/resources/data/forge/tags/worldgen/biome/is_dry.json

```diff
@@ -0,0 +1,16 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:desert",
+    "minecraft:nether_wastes",
+    "minecraft:the_end",
```

Comment:
is there a reason only one end biome is dry but not any of the others? maybe we should add all end biomes to this tag.